#!/usr/bin/env python
PACKAGE = "rgb_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("color_b_lower", int_t, 0, "Blue lower bound", 0, 0, 255)
gen.add("color_b_upper", int_t, 0, "Blue upper bound", 255, 0, 255)
gen.add("color_g_lower", int_t, 0, "Green lower bound", 0, 0, 255)
gen.add("color_g_upper", int_t, 0, "Green upper bound", 255, 0, 255)
gen.add("color_r_lower", int_t, 0, "Red lower bound", 0, 0, 255)
gen.add("color_r_upper", int_t, 0, "Red upper bound", 255, 0, 255)

exit(gen.generate(PACKAGE, "rgb_filter", "RGBFilter"))
